lib/PageAgent.js

```diff
@@ -46,6 +46,12 @@ extend(PageAgent.prototype, {
       this.emit('resource-tree');
     }.bind(this);
 
+    if (!this._debuggerClient.isHostMachine) {
+        return this._debuggerClient.once(
```

Comment:
Please add a comment explaining what's the purpose of this branch, i.e. skip building a list of source files from the filesystem because we are running on a different machine.